fuzzing/oss_fuzz_build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -eu
```

Comment:
It actually looks like to me that this could just be it's own gradle module with a dependency on sigstore-java? Delegate to gradle to actually do the right thing for the build?

Reply:
I don't know how the jazzer_api is injected in (presumably in the build container?)

Reply:
you could use the application plugin to create runnable targets?

Reply:
I think it could be a gradle module as long as we get the flags from the OSS-Fuzz environment. I am happy to make it a TODO, but would prefer to get the fuzzers running the current way first. Is the current build script a blocker to you?

Reply:
kk sounds good.